src/SmartComponents/redux/reducers.js

```diff
@@ -18,3 +18,33 @@ export function composeReducer(state = { }, action) {
             return state;
     }
 }
+
+const initialPendingComposeState = {
+    release: {
+        arch: 'x86_64',
+        release: 'rhel-8',
```

Comment:
Maybe we could rename one of the releases so it isn't `release.release`? `release.distro`?